### PR TITLE
Fix confirm attribute usage in slim templates

### PR DESCRIPTION
**Pull request by ilyasgaraev**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+- Fix `confirm` attribute usage in slim templates
```
Comment (timurvafin) [inline CHANGELOG.md]: Don't think we should mention this minor patch in CHANGELOG :)

**Comment by ilyasgaraev** [comment]
@timurvafin review please

**Comment by artembiserov** [comment]
@ilyasgaraev I think you will have some warning like this:

Name: actionpack
Version: 4.2.5
Advisory: CVE-2015-7576
Criticality: Unknown
URL: https://groups.google.com/forum/#!topic/rubyonrails-security/ANv0HDHEC3k
Title: Timing attack vulnerability in basic authentication in Action Controller.
Solution: upgrade to ~> 5.0.0.beta1.1, ~> 4.2.5.1, ~> 4.1.14.1, ~> 3.2.22.1

Please, update rails , I think in different pr

**Comment by timurvafin** [comment]
@ilyasgaraev btw, are these scaffold templates useful at all right now?

**Comment by ilyasgaraev** [comment]
@timurvafin I fixed templates, even though they are probably not used anywhere at all. But I would prefer to not have mistakes there at all.

**Comment by ilyasgaraev** [comment]
@artembiserov updated in https://github.com/fs/rails-base/pull/409

**Pull request merged**